src/main/webapp/user-page.jsp

```diff
@@ -93,16 +90,18 @@ String uploadUrl = blobstoreService.createUploadUrl("/my-form-handler"); %>
       <button>Submit</button>
     </form>
 
-    
+
 
     <div id="message-container">Loading...</div>
 
     <script>
```

Comment:
does it matter if this script calls each function separately rather than the buildUI() function like it currently does in master?

Reply:
Nope. I updated it to buildUI()